api/graphql/schema.graphql

```diff
@@ -1909,6 +1922,10 @@ type Mutation {
   updateUserSkill(id: ID!, userSkill: UpdateUserSkillInput @spread): UserSkill
     @update
     @can(ability: "update", find: "id")
+  updateUserSkillRankings(
+    userId: UUID!
```

Comment:
Should we validate ids? Like, should we validate that this is a real userId, and that the skillRank arrays are all valid skill ids?
I know its not something we've been doing, but I just ran into an error during testing where I mis-pasted a uuid. If our client code did something like this in production, the user would get an opaque Internal Server Error.
@esizer @petertgiles thoughts?

```
"SQLSTATE[23503]: Foreign key violation: 7 ERROR:  insert or update on table \"user_skills\" violates foreign key constraint \"user_skills_skill_id_foreign\"\nDETAIL:  Key (skill_id)=(4526364b-196a-415a-a362-2305a9a14e02) is not present in table \"skills\". (Connection: pgsql, SQL: insert into \"user_skills\" (\"skill_id\", \"user_id\", \"updated_at\", \"created_at\") values (4526364b-196a-415a-a362-2305a9a14e02, 8259bf4b-4fb5-4f59-b55e-d79ec6dfd09a, 2023-09-05 14:30:03, 2023-09-05 14:30:03) returning \"id\")"
```

Reply:
Yeah, I think this could be useful. I wonder if laravel has a built in rule for this, or we could make a `ModelExists` rule or something 🤔